src/Google.Cloud.Functions.Hosting/HostingInternals.cs

```diff
@@ -84,6 +88,25 @@ internal static IApplicationBuilder ConfigureApplication(WebHostBuilderContext c
                 });
         }
 
+        /// <summary>
+        /// Validates that the startup classes which would be used for the finally-selected function are the
+        /// same as the startup classes we *actually* used. This method is only called when the app is started
```

Comment:
...are the same and in the same order...

Reply:
Done.